prusti/tests/verify/pass/quick/marker-traits.rs

```diff
@@ -1,6 +1,7 @@
-extern crate prusti_contracts;
+#![feature(register_tool)]
+#![register_tool(prusti)]
 
-#[invariant="self.d1 == self.d2"]
+#[trait_invariant(self.d1 == self.d2)]
```

Comment:
Could you add a comment that `trait_invariant` currently is not even parsed so that people do not get confused?

cc @cmatheja